viewer/packages/data-providers/src/image-360-data-providers/descriptor-providers/datamodels/cdm/get360CdmCollectionsQuery.ts

```diff
@@ -0,0 +1,106 @@
+/*!
+ * Copyright 2025 Cognite AS
+ */
+
+import {
+  COGNITE_360_IMAGE_COLLECTION_SOURCE,
+  COGNITE_360_IMAGE_SOURCE,
+  COGNITE_360_IMAGE_STATION_SOURCE,
+  MAX_DMS_QUERY_LIMIT
+} from '../../../../utilities/constants';
+import { DMInstanceRef } from '@reveal/utilities';
+import { CORE_DM_IMAGE_360_PROPERTIES_LIST } from '../../../cdm/queryProperties';
+
+function createCollectionsQuery(collectionRefs: DMInstanceRef[]) {
+  const collectionFilters = collectionRefs.map(ref => ({
+    and: [
+      {
+        equals: {
+          property: ['node', 'externalId'] as const,
+          value: ref.externalId
+        }
+      },
+      {
+        equals: {
+          property: ['node', 'space'] as const,
+          value: ref.space
+        }
+      }
+    ]
+  }));
```

Comment:
I think you can use the `instanceReferences` filter here, which may help us avoid maxing out the number of and/or clauses in the query

<img width="1446" height="1586" alt="Image" src="https://github.com/user-attachments/assets/53baf86e-28f8-44bf-82d6-f3273e893ffe" />